library/src/com/squareup/timessquare/CalendarPickerView.java

```diff
@@ -814,6 +934,7 @@ private static boolean sameMonth(Calendar cal, MonthDescriptor month) {
   }
 
   private boolean isDateSelectable(Date date) {
+    if (isCheckDayInSelectableCals && !containsDate(selectableCals, date)) return false;
```

Comment:
We'd use `{` and `}` here or just fold the conditional into a single `return` statement.